### ISSUE TITLE
Intl.DateTimeFormat timeZoneName not supported by IE11

### ISSUE THREAD

**Issue by tjcrowder** [mdn/browser-compat-data#6400]
We should add a row to [the `Intl.DateTimeFormat` constructor page's data](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) saying that `timeZoneName` isn't supported by IE11. On IE11, this code just produces `12:49`:

```js
document.body.insertAdjacentText(
    "beforeend",
    new Intl.DateTimeFormat(undefined, {hour: "2-digit", minute: "2-digit", timeZoneName: "long", hour12: false}).format(new Date())
);
```

(Same with `"short"` instead of `"long"`.) In contrast, Firefox, Chrome, and iOS Safari include a timezone name (of some kind).

I don't have time to do the update and PR right now, but I'll try to in the next couple of weeks.

**Comment by stof** [comment]
Lots of options are missing in the data for the Intl.DateTimeFormat.

**Comment by ddbeck** [comment]
Yeah, we're missing a lot of `Intl` options data. Related issues include #6046 and #6525. I'd like to see us improve on this, but I think it's a bit of a project. If someone is interested in guiding this at all, I'd welcome a tracking issue to at least list all of the options currently specified—then we could at least have a sense of the size of the problem and its progress.

**Comment by ddbeck** [comment]
Right. We describe the options on the wiki, but we don't have subfeatures on the constructor showing which options are supported.

**Comment by jculver050** [comment]
Close
